src/XMLimport.h

```diff
@@ -50,7 +50,7 @@ class XMLimport : public QXmlStreamReader
 public:
     XMLimport(Host*);
 
-    bool importPackage(QIODevice* device, QString packageName = QString(), int moduleFlag = 0);
+    bool importPackage(QIODevice*, QString packageName = QString(), int moduleFlag = 0, QString* pVersionString = Q_NULLPTR);
```

Comment:
Why not ``nullptr``? We enforce C++11 and a lot more tools will understand ``nullptr`` than Qt's macro that's only useful if you are allowing < C++11.